python/ray/tune/logger/__init__.py

```diff
@@ -8,6 +8,7 @@
 from ray.tune.logger.json import JsonLogger, JsonLoggerCallback
 from ray.tune.logger.noop import NoopLogger
 from ray.tune.logger.tensorboardx import TBXLogger, TBXLoggerCallback
+from ray.tune.logger.aim import AimLoggerCallback
```

Comment:
We shouldnt import this here; it should be done lazily

Reply:
I think it's okay to import here - we have a try/catch for importing the aim module stuff. Will only error out when instantiating an `AimLoggerCallback`.

Reply:
This isn't consistent with what we do with the rest of the 3rd party Callbacks. Can we keep it consistent?

Reply:
I see. So TBX is not considered "3rd party" since it's default.

Reply:
Gonna remove this import then.